src/lib/Scenes/Artwork/Components/CommercialButtons/CommercialButtons.tsx

```diff
@@ -61,7 +63,7 @@ export class CommercialButtons extends React.Component<CommercialButtonProps> {
       return <BuyNowButton artwork={artwork} editionSetID={this.props.editionSetID} />
     } else if (isOfferable) {
       return <MakeOfferButton artwork={artwork} editionSetID={this.props.editionSetID} />
-    } else if (isInquireable) {
+    } else if (isInquireable && !artworkSaleEnded) {
```

Comment:
To replicate the logic on desktop, we can just use `!artwork.isInAuction` here!

https://github.com/artsy/reaction/blob/6f883d2603287af8dbed5671f24e889a614dc1dd/src/Apps/Artwork/Components/ArtworkSidebar/index.tsx#L51

Reply:
It'll accomplish the same thing but seems technically more correct as we never want to show the `Contact Gallery` button if an artwork is in an auction.

Reply:
there is also this guy inside the `ArtworkSidebarBidAction`: https://github.com/artsy/reaction/blob/master/src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx#L92 🤔